### PR TITLE
FIX-1030: Add a light theme to supported themes

### PR DESCRIPTION
**Pull request by evargast**
Closes [1030](https://github.com/adobe/react-spectrum/issues/1030)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [x] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [x] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 📝 Test Instructions:

No test instructions, added another theme that leverages Adobe's Lightest CSS guidelines for the `light` color scheme. 

(defaultTheme uses `light` instead of `lightest` which gives it a slightly grey base-color)

## 🧢 Your Project:

Adobe Commerce Cloud

### CODE REVIEW

File: packages/@react-spectrum/theme-light/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@react-spectrum/theme-light",
+  "version": "3.1.0",
```
Comment (evargast) [inline packages/@react-spectrum/theme-light/package.json:3]: Wasn't 100% about the version, both default and dark themes are in `3.1.0` so made it match, down to change it to whatever though!

Comment (devongovett) [reply packages/@react-spectrum/theme-light/package.json:3]: This is fine. Our release scripts will take care of correcting it when it goes out! 😄

**Review by devongovett — APPROVED** [review]
Thanks! We'll need to remember to add this to the [docs](https://react-spectrum.adobe.com/react-spectrum/theming.html#how-themes-are-defined), and also to the monopackage before the next release. 😄

**Comment by evargast** [comment]
> Thanks! We'll need to remember to add this to the [docs](https://react-spectrum.adobe.com/react-spectrum/theming.html#how-themes-are-defined), and also to the monopackage before the next release. 😄

Ah, I should've added that! Update incoming, stay tuned

**Comment by dannify** [comment]
Thanks @evargast because the docs are included in this pr we will merge it when it gets released so the docs are published early. Alternatively we could split the pr and have the docs separate. I don't mind either way, I just wanted to give you an update.

**Comment by Eduardo-Vargas** [comment]
Thanks @dannify ! I'm fine either way, luckily I'm not in need of this PR atm, just thought it would be useful to contribute it :)

**Pull request merged**